package.json

```diff
@@ -51,7 +51,7 @@
         "verdaccio": "docker run --rm -i -p 4873:4873 0xorg/verdaccio"
     },
     "config": {
-        "contractsPackages": "@0x/contracts-erc20  @0x/contracts-test-utils @0x/contracts-utils @0x/contracts-zero-ex @0x/contracts-treasury",
+        "contractsPackages": "@0x/contracts-erc20 @0x/contracts-test-utils @0x/contracts-utils @0x/contracts-zero-ex @0x/contracts-treasury",
```

Comment:
this one breaks `artifacts_update` in `contract-artifacts`.
I didn't update the artifacts/wrappers because we're inbetween migrations right now